ansible/roles/telegraf/templates/telegraf.conf.j2

```diff
@@ -0,0 +1,262 @@
+#    Counters = [
+#      "Disk Read Bytes/sec",
+#      "Disk Write Bytes/sec",
+#      "Current Disk Queue Length",
+#      "Disk Reads/sec",
+#      "Disk Writes/sec",
+#      "% Disk Time",
+#      "% Disk Read Time",
+#      "% Disk Write Time",
+#    ]
+#    Measurement = "win_diskio"
+
+#  [[inputs.win_perf_counters.object]]
+#    ObjectName = "Network Interface"
+#    Instances = ["*"]
+#    Counters = [
+#      "Bytes Received/sec",
+#      "Bytes Sent/sec",
+#      "Packets Received/sec",
+#      "Packets Sent/sec",
+#      "Packets Received Discarded",
+#      "Packets Outbound Discarded",
+#      "Packets Received Errors",
+#      "Packets Outbound Errors",
+#    ]
+#    Measurement = "win_net"
+
+#  [[inputs.win_perf_counters.object]]
+#    ObjectName = "System"
+#    Counters = [
+#      "Context Switches/sec",
+#      "System Calls/sec",
+#      "Processor Queue Length",
+#      "System Up Time",
+#    ]
+#    Instances = ["------"]
+#    Measurement = "win_system"
+#    # Set to true to include _Total instance when querying for all (*).
+#    #IncludeTotal=false
+
+  [[inputs.win_perf_counters.object]]
+    # Example query where the Instance portion must be removed to get data back,
+    # such as from the Memory object.
+    ObjectName = "Memory"
+    Counters = [
+      "Available Bytes",
+    ]
+    # Use 6 x - to remove the Instance bit from the query.
+    Instances = ["------"]
+    Measurement = "win_mem"
+    # Set to true to include _Total instance when querying for all (*).
+    #IncludeTotal=false
+# Not used stats
+#      "Cache Faults/sec",
+#      "Demand Zero Faults/sec",
+#      "Page Faults/sec",
+#      "Pages/sec",
+#      "Transition Faults/sec",
+#      "Pool Nonpaged Bytes",
+#      "Pool Paged Bytes",
+#      "Standby Cache Reserve Bytes",
+#      "Standby Cache Normal Priority Bytes",
+#      "Standby Cache Core Bytes",
+
+#  [[inputs.win_perf_counters.object]]
+#    # Example query where the Instance portion must be removed to get data back,
+#    # such as from the Paging File object.
+#    ObjectName = "Paging File"
+#    Counters = [
+#      "% Usage",
+#    ]
+#    Instances = ["_Total"]
+#    Measurement = "win_swap"
+#
+#  [[inputs.win_perf_counters.object]]
+#    ObjectName = "Network Interface"
+#    Instances = ["*"]
+#    Counters = [
+#      "Bytes Sent/sec",
+#      "Bytes Received/sec",
+#      "Packets Sent/sec",
+#      "Packets Received/sec",
+#      "Packets Received Discarded",
+#      "Packets Received Errors",
+#      "Packets Outbound Discarded",
+#      "Packets Outbound Errors",
+#    ]
+
+
+
+# Windows system plugins using WMI (disabled by default, using
+# win_perf_counters over WMI is recommended)
+
+# # Read metrics about cpu usage
+# [[inputs.cpu]]
+#   ## Whether to report per-cpu stats or not
+#   percpu = true
+#   ## Whether to report total system cpu stats or not
+#   totalcpu = true
+#   ## Comment this line if you want the raw CPU time metrics
+#   fielddrop = ["time_*"]
```

Comment:
I think that these `[[inputs.*]]` sections (without `win_perf_counters`) could be remove